docs/api/v1/openapi.yml

```diff
@@ -952,10 +952,8 @@ paths:
         - Variables
       summary: Update a variable
       description: Update a variable from your instance.
-      parameters:
-        - $ref: '#/components/parameters/variableId'
```

Comment:
Does it make sense to remove this? As far as I can tell, the variable ID is still required in the path.

Reply:
I'm not sure why this has been removed. The original PR didn't touch variables so I wonder if we've just not copied across the generated YAML for a while?

This PR? https://github.com/n8n-io/n8n/pull/15315